Library/Homebrew/exceptions.rb

```diff
@@ -25,9 +25,9 @@ def initialize(name)
 class FormulaValidationError < StandardError
   attr_reader :attr
 
-  def initialize(attr, value)
+  def initialize(full_name, attr, value)
```

Comment:
It may be worth to just call this variable as `formula` and store it as instance variable.